spacelift/resource_context.go

```diff
@@ -98,11 +99,12 @@ func resourceContextUpdate(d *schema.ResourceData, meta interface{}) error {
 		variables["description"] = toOptionalString(description)
 	}
 
-	if err := meta.(*Client).Mutate(&mutation, variables); err != nil {
-		return errors.Wrap(err, "could not update context")
-	}
+	var acc multierror.Accumulator
 
-	return resourceContextRead(d, meta)
+	acc.Push(errors.Wrap(meta.(*Client).Mutate(&mutation, variables), "could not update context"))
+	acc.Push(resourceContextRead(d, meta))
```

Comment:
Maybe wrap this with "could not read the current state"?

Reply:
@marcinwyszynski I've changed it.